CHANGELOG.md

```diff
@@ -15,6 +15,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 - Stylize prompt to create new project or tag (#310).
 - Aggregate calculates wrong time if used with `--current` (#293)
+- `watson start` now correctly checks if project is empty (#322)
```

Comment:
I would be consistent with the rest of the changelog and talk about the `start` command instead of `watson start`.

Reply:
Agree. Changes applied.